From efbb25c7f71d06a53a06324c0e6d53756d286f1c Mon Sep 17 00:00:00 2001
From: Brion Vibber <brion@pobox.com>
Date: Fri, 17 Oct 2014 22:04:20 -0700
Subject: [PATCH] Fix for OS X 10.9 and 10.10 builds

Can't use undefined symbols from GL libraries!
cogl has updated and hopefully this special case not needed
---
 clutter-gst/clutter-gst-video-sink.c | 11 -----------
 1 file changed, 11 deletions(-)

diff --git a/clutter-gst/clutter-gst-video-sink.c b/clutter-gst/clutter-gst-video-sink.c
index 63867d4..2fec757 100644
--- a/clutter-gst/clutter-gst-video-sink.c
+++ b/clutter-gst/clutter-gst-video-sink.c
@@ -713,7 +713,6 @@ _string_array_to_char_array (char *dst, const char *src[])
 }
 #endif
 
-#if !defined (HAVE_CLUTTER_OSX)
 static gint
 get_n_fragment_texture_units (void)
 {
@@ -732,16 +731,6 @@ get_n_fragment_texture_units (void)
 
   return n;
 }
-#else
-static gint
-get_n_fragment_texture_units (void)
-{
-  gint n_texture_units;
-
-  glGetIntegerv (GL_MAX_COMBINED_TEXTURE_IMAGE_UNITS, &n_texture_units);
-  return n_texture_units;
-}
-#endif
 
 static CoglHandle
 _create_cogl_program (const char *source)
-- 
1.9.3 (Apple Git-50)

